core/src/test/java/com/microsoft/applicationinsights/internal/util/DateTimeUtilsTests.java

```diff
@@ -0,0 +1,60 @@
+package com.microsoft.applicationinsights.internal.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.text.ParseException;
+import java.util.Date;
+
+/**
+ * Created by yonisha on 5/26/2015.
+ */
+public class DateTimeUtilsTests {
+
+    @Test
+    public void testParseRoundTripDateString() throws ParseException {
+        final String dateStr = "2015-05-26T07:00:14+01:00";
+        String reversedDateStr = parseRoundTripDateString(dateStr);
```

Comment:
`parseRoundTripDateString` returns a Date rather than a string so it would be better to explicitly call toString() here.